### PR TITLE
Calendar style

### PR DESCRIPTION
**Pull request by lilyosah**
#73

### CODE REVIEW

File: spec/features/calendar_feature_spec.rb [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
-    Task.create!(title: 'SWE iteration 2', start: DateTime.new(2021, 11, 8),  due: DateTime.new(2021, 11, 21))
-    Task.create!(title: 'SWE iteration 3', start: DateTime.new(2021, 11, 22),  due: DateTime.new(2021, 12, 5))
-    Task.create!(title: 'Physics HW', description: "boring", start: DateTime.new(2021, 11, 3),  due: DateTime.new(2021, 11, 4))
+    Task.create!(title: 'SWE iteration 2', start: DateTime.new(2021, 11, 8), due: DateTime.new(2021, 11, 21))
+    Task.create!(title: 'SWE iteration 3', start: DateTime.new(2021, 11, 22), due: DateTime.new(2021, 12, 5))
+    Task.create!(title: 'Physics HW', description: 'boring', start: DateTime.new(2021, 11, 3),
+                 due: DateTime.new(2021, 11, 4))
     # Task.create!(title: 'Write Capybara tests', description: "not boring", start: DateTime.new(2021, 11, 3),  due: DateTime.new(2021, 11, 3))
-    Task.create!(title: 'Put away christmas decorations', due: DateTime.new(2022, 01, 7))
+    Task.create!(title: 'Put away christmas decorations', due: DateTime.new(2022, 0o1, 7))
```
Comment (mghilea) [inline spec/features/calendar_feature_spec.rb:17]: '0o1' is probably a mistake

**Review by mghilea — COMMENTED** [review]
Looks good, just check if the bin folder needs to be merged into main

**Comment by lilyosah** [comment]
@mghilea Seems like bin is added by bundler and the stuff was already committed 27 days ago, there are only changes because I ran rubocop which reformatted things. Would u mind approving?

**Review by mghilea — APPROVED** [review]
Looks good!

**Pull request merged**